### ISSUE TITLE
Add RTL button to editor

### ISSUE THREAD

**Issue by rfnbnkr** [gollum/gollum#812]
Hi. i can write any LTR language. I want write document in RTL lang. problem solved with add "direction: rtl;" to editor.css . But i want add RTL button to editor that i can choose text direction.
sorry for my english

**Comment by dometto** [comment]
We will move to the `ace` editor by default in `5.x`. RTL support in `ace` is blocked on https://github.com/ajaxorg/ace/issues/1429

**Comment by dometto** [comment]
Closing in favor of #1314. @rfnbnkr can you let us know there whether things have improved in the `5.x` branch?

**Comment by dometto** [comment]
Anyone watching this thread interested in RTL support, please have a look at https://github.com/gollum/gollum/pull/1350 and let us know whether the implementation there gives you what you need!

/cc @rfnbnkr
